Modules/Axis/Sources/Axis/Log/Log.swift

```diff
@@ -51,6 +47,13 @@ public final class Logger {
         self.appenders.append(contentsOf: appenders)
         self.name = name
     }
+
+    fileprivate static let logTimeFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        let format = DateFormatter.dateFormat(fromTemplate: "HH:mm:ss.SSS", options: 0, locale: .current)
```

Comment:
Could we also include the day/month/year and locale here? I think that would be useful. Ideal date IMO: `10/06/16 12:42:10.123 GMT-8`.

Note - I'm not sure if `DateFormatter` supports figuring out whether the month should go before or in-between the day and year (in the US, we have the day in the middle).

I also think we should use the 24-hour system regardless of the locale, so the code you have now (`HH`) is good.